app/controllers/location_controller.rb

```diff
@@ -599,8 +599,8 @@ def create_location
           if (user = User.safe_find(@set_user))
             user.location = @location
             user.save
-            redirect_to(controller: :observer,
-                        action: :show_user,
+            redirect_to(controller: :users,
+                        action: :show,
```

Comment:
Why isn't this user_path(@set_user) as elsewhere above?

Reply:
Changed to user_path(@set_user.id)